tests/utils/test_uri.py

```diff
@@ -293,25 +293,25 @@ def test_append_to_uri_path_preserves_uri_schemes_hosts_queries_and_fragments():
             (
                 "dbscheme+dbdriver://root:password//path/#myfragmentwith$pecial@",
                 "subpath/anotherpath",
-                "dbscheme+dbdriver://root:password//path/subpath/anotherpath#myfragmentwith$pecial@",  # noqa
+                "dbscheme+dbdriver://root:password//path/subpath/anotherpath#myfragmentwith$pecial@",  # pylint: disable=line-too-long
```

Comment:
Do we still need  the pylint disable comment given we have the regex to match URL-like lines?

Reply:
Yes because the regex only matches http-URL-like lines.

Reply:
How about we extend it to match "://" (ignoring the schema)?

Reply:
I feel like that's too broad.

Reply:
I feel like `"://"` is too broad.